### PR TITLE
Don't show opt-in/ft.com buttons on clippings pages for Next users

### PR DESCRIPTION
**Pull request by keirog**

### CODE REVIEW

**Comment by laurieboyes** [comment]
:+1:

File: server/controllers/product-tour.js [hunk 1]
```diff
@@ -216,6 +216,7 @@ module.exports = function (req, res) {
 		viewModel.layout = 'vanilla';
 		viewModel.clippingsLinks = getClippingsLinks(req);
 		viewModel.isLoggedOut = false;
+		viewModel.isNextUser = req.cookies.FT_SITE === 'NEXT';
```
Comment (commuterjoy) [inline server/controllers/product-tour.js]: I hope cookies are removed from the backend by the CDN, so this probably will break the caching?

Comment (phamann) [inline server/controllers/product-tour.js]: @commuterjoy IIRC this page is not being served on `next.ft.com` but only the interstitial `clippings.ft.com` which will not be CDN'd. The Cookies are only used to determine whether the user should bypass interstitial messaging etc.

Will check with @keirog to confirm.

Comment (keirog) [inline server/controllers/product-tour.js]: @commuterjoy Yep, @phamann's right (ta)

Comment (matthew-andrews) [inline server/controllers/product-tour.js]: worth adding a comment to this effect?

Comment (commuterjoy) [inline server/controllers/product-tour.js]: ok, thanks.

**Pull request merged**